elements/bulbs-video/components/revealed.js

```diff
@@ -93,6 +93,10 @@ export default class Revealed extends React.Component {
       twitterHandle: this.props.twitterHandle,
     };
 
+    if (this.props.noEndcard) {
+      delete playerOptions.pluginConfig.endcard;
```

Comment:
This feels like an anti-pattern. If we don't want an endcard, couldn't we not pass in the endcard config?